embedding/embedlite/embedshared/EmbedLiteWindowBaseChild.cpp

```diff
@@ -89,7 +91,13 @@ void EmbedLiteWindowBaseChild::ActorDestroy(ActorDestroyReason aWhy)
 
 mozilla::ipc::IPCResult EmbedLiteWindowBaseChild::RecvDestroy()
 {
+  if (initialized == false) {
+    destroyAfterInit = true;
+    return IPC_OK();
+  }
+
   LOGT("destroy");
+  mWidget->Destroy();
```

Comment:
Given that https://github.com/sailfishos/gecko-dev/pull/9/files#r706130282 we'd return earlier on the CreateWidget then this should be guarded with

```
if (mWidget) {
    mWidget->Destroy();
}
```